ted2zim/entrypoint.py

```diff
@@ -110,10 +107,23 @@ def main():
         "--debug", help="Enable verbose output", action="store_true", default=False
     )
 
+    parser.add_argument(
+        "--version",
+        help="Display scraper version and exit",
+        action="version",
+        version=SCRAPER,
+    )
+
     args = parser.parse_args()
     logger.setLevel(logging.DEBUG if args.debug else logging.INFO)
 
     try:
+        if args.max_videos_per_topic < 1:
+            raise ValueError(
+                "Maximum number of videos to scrape per topic must be greater than or equal to 1"
+            )
+        if not args.topics:
```

Comment:
I don't think this is possible, as `topics` is a required arg.

Reply:
It's required as it would allow to fail if someone passes an empty string as "".